config/cartographer.lua

```diff
@@ -52,4 +52,4 @@ POSE_GRAPH.constraint_builder.global_localization_min_score = 0.65
 POSE_GRAPH.optimization_problem.huber_scale = 1e2
 POSE_GRAPH.optimize_every_n_nodes = 30
 
-return options
+return options
```

Comment:
Missing newline. Please rebase on top of the main development branch and then `pre-commit run all-files` which will pick up any files in this PR.